visma/io/checks.py

```diff
@@ -48,8 +48,11 @@ def isNumber(term):
         if term[0] == '-':
             x += 1
             while x < len(term):
-                if (term[x] < '0' or term[x] > '9') and (dot != 0 or term[x] != '.'):
-                    return False
+                try:
```

Comment:
It is a good practice to avoid the use of try-except as much as possible. Since it is always a TypeError caused due to comparison between float and str, the following can be used instead of try-except: 
```suggestion
                if (str(term[x]) < '0' or str(term[x]) > '9') and (dot != 0 or str(term[x]) != '.'):
                    return False
```
Similar change can be made in lines 65-69.